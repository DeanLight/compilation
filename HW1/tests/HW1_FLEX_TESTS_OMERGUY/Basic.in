obj
endobj
obj
endobj
obj obj obj endobj obj
[ ] ] [ [[]]
<< >> >> << <<<<>>>>
% This is a comment, a very good one. % Commentception
% Another comment, should be ok with all chars: !"#$%&'()*+,-.0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\]^_`abcdefghijklmnopqrstuvwxyz{|}~	
true false false true true true false true false
false
true
1 +12 -123 123 +123 0 +0 -0 000000002
.1 +.12 -0.123 1.23 +123.02 +123. 0 +0.0 -0.0 000.000002000 -.123200
(Ground Tom) (To) (Maj. CTRL)
(Ground Major\nTo Control Tom\r\n)
(Parents \(\) HAD escaped!!! CALL DA policia)
( git me baby \
1 more t1m3) ()
( \
\
\
\
\
\
)
(\(\))
(ESCAPING DA va'hadat mishma'hat \r\n\t\b\f\(\)\\)
(\104\157\040\171\157\165\040\
\154\151\153\145\040\141\166\157\143\141\144\157\077)
( !"#$%&'*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[]^_`abcdefghijklmnopqrstuvwxyz{|}~)
<4d 6f 74 6f 72 20 61 6c 6c 20 66 72 65 65 7a 65 20 66 75 6e 63 74 69 6f 6e 73>
<4D 6F 74 6F 72 20 61 6C 6C 20 66 72 65 65 7A 65 20 66 75 6E 63 74 69 6F 6E 73>
<4d 6F 74 6F 72 20 61 6C 6C 20 66 72 65 65 7A 65 20 66 75 6E 63 74 69 6F 6e 73>
<>
/Hello /my/name/Yaakov/whats/your/name/my/name/is/Ernnio
/12/234123/523524/o
stream
xœÕ]Y7’~ ÿP/TÍ¢S<“IÀhlw«ÛÐ@Û’û`ïƒÖ–m
¼Z¤ÁìÏ_ž™<ƒ¬,¶0‹w««"øF>{óö××ÿôÇ»Ÿ>¾úêYð×³o?þüîãû¿®¯oŸßÏ¾ÓßÜ½x~@Ï^¾U_{wõâåéúú 	nß<}òì0Ÿä|xóËÓ'ø€Ôÿð09qzŒNt9¼ùoE÷õkŒ¿~zúDýÔ?¾~úä‡¯¹a×WRÿ–×Lýº¯¯¸þóîšŸ*"óç½ûÍÕgB?Œ¢êß7×WDÓ-ú;ûÈÍõÞüùé“{%äáþ¥Žþñl—ö¸¦ý¦/™õýJKvýæ¯c uó“Æb‘ù)&çAÄ$¬8?Xy®œi°3®±Ú
6ß9³_¹¯ý‹lv­4”’,CÑ<1šŠ‚YÚØ>»°"GÕ	¸ ü¸ñÀKC@ŠÛ[<í\dË&,U[QbËÅ´¤Úú‘dfð­¼6óš»AvY0èÔ×ã$\Š23%“‡š£XFÓ‹¦¡jLÇÓëøðû‰ßäñOCF6VÜÒ”'JÍ³±+n®ÏWÇëøƒ_’Å5Fºï‘s¸ù'ZjÌxÙì³A•)›T!'O’fäÄ®ow×FçF”;Q™1:[×Ä¶eáçöÓÚi†»ýÎ<ÄüŸ#G:®z‚Uaí#…Ú’€11áÃ,ñ4ÏÛÄÃ×Q·ûeÁ™oý{¾Æ$YIl?¨Ã’3ûÑ@›²¶F8Öh°Iysnq³ÊÏ³œ$ÛFq2e²Ô†6lZ¦;h	þ[r-JØÁ÷—oÙ†ä
%L›§!á±=£	gß¯²6Û9ÎÝ[DÎæ	Ñ‚jäDƒ“¤õç¦“ O‚Gm;ÑãûßÕ·(oño?}ä+DÛ=Èeš3M‡Â\ôÂBuˆÈ:„ÚQˆí`0t$PÂ²(
Ï:»Ì7ß¨fƒ¢Î>Œé\Ru¼åžfDNl>ÌjæaG›©¦]½Ó°ÓTm¯x¸~˜5"\	æð[ë:W?}×¶iÚ)§>“@èÃšŠê'…­n¾dKÚ6Ò"‚ -Âè¹BJ¬
endstream
null null null null null
null