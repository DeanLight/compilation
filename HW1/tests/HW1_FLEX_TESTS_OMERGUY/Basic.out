1 OBJ obj
2 ENDOBJ endobj
3 OBJ obj
4 ENDOBJ endobj
5 OBJ obj
5 OBJ obj
5 OBJ obj
5 ENDOBJ endobj
5 OBJ obj
6 LBRACE [
6 RBRACE ]
6 RBRACE ]
6 LBRACE [
6 LBRACE [
6 LBRACE [
6 RBRACE ]
6 RBRACE ]
7 LDICT <<
7 RDICT >>
7 RDICT >>
7 LDICT <<
7 LDICT <<
7 LDICT <<
7 RDICT >>
7 RDICT >>
8 COMMENT % This is a comment, a very good one. % Commentception
9 COMMENT % Another comment, should be ok with all chars: !"#$%&'()*+,-.0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\]^_`abcdefghijklmnopqrstuvwxyz{|}~	
10 TRUE true
10 FALSE false
10 FALSE false
10 TRUE true
10 TRUE true
10 TRUE true
10 FALSE false
10 TRUE true
10 FALSE false
11 FALSE false
12 TRUE true
13 INTEGER 1
13 INTEGER +12
13 INTEGER -123
13 INTEGER 123
13 INTEGER +123
13 INTEGER 0
13 INTEGER +0
13 INTEGER -0
13 INTEGER 000000002
14 REAL .1
14 REAL +.12
14 REAL -0.123
14 REAL 1.23
14 REAL +123.02
14 REAL +123.
14 INTEGER 0
14 REAL +0.0
14 REAL -0.0
14 REAL 000.000002000
14 REAL -.123200
15 STRING Ground Tom
15 STRING To
15 STRING Maj. CTRL
16 STRING Ground Major
To Control Tom

17 STRING Parents () HAD escaped!!! CALL DA policia
19 STRING  git me baby 1 more t1m3
19 STRING 
26 STRING  
27 STRING ()
28 STRING ESCAPING DA va'hadat mishma'hat 
	()\
30 STRING Do you like avocado?
31 STRING  !"#$%&'*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[]^_`abcdefghijklmnopqrstuvwxyz{|}~
32 STRING Motor all freeze functions
33 STRING Motor all freeze functions
34 STRING Motor all freeze functions
35 STRING 
36 NAME /Hello
36 NAME /my
36 NAME /name
36 NAME /Yaakov
36 NAME /whats
36 NAME /your
36 NAME /name
36 NAME /my
36 NAME /name
36 NAME /is
36 NAME /Ernnio
37 NAME /12
37 NAME /234123
37 NAME /523524
37 NAME /o
44 STREAM x≈ì√ï]Y¬è7‚Äô~¬†√øP/T√ç¬¢S<‚ÄúI√Ählw¬´√õ√ê@√õ‚Äô√ª`√Ø∆í√ñ‚Äìm
¬ºZ¬è¬§√Å√¨√è_≈æ‚Ñ¢<∆í¬¨,¬∂0‚Äπw¬´¬´"¬è√∏F>{√≥√∂√ó√ó√ø√¥√á¬ª≈∏>¬æ√∫√™Y√∞√ó¬≥o?√æ√º√Æ√£√ª¬ø¬Æ¬Øo≈∏√ü√è¬æ√ì√ü√ú¬Ωx~@√è^¬æU_{w√µ√¢√•√©√∫√∫¬†	n√ü<}√≤√¨0≈∏√§|x√≥√ã√ì'√∏‚Ç¨√î√ø√∞¬Å09qz≈íNt9¬º√πoE√∑√µk≈í¬ø~z√∫D√Ω√î?¬æ~√∫√§‚Ä°¬Ø¬πa√óWR√ø‚Äì√óL√Ω¬∫¬ù¬Ø¬Ø¬∏√æ√≥√Æ≈°≈∏*"√≥√ß¬Ω√ª√ç√ïgB?≈í¬¢√™√ü7√óWD√ì-√∫;√ª√à√ç√µ√û√º√π√©‚Äú{%√§√°√æ¬•≈Ω√æ√±l‚Äî√∂¬∏¬¶√Ω¬¶/‚Ñ¢√µ√ΩJKv√Ω√¶¬Øc u√≥‚Äú√Üb‚Äò√π)&√ßA√Ñ$¬¨8?Xy¬Æ≈ìi¬∞3¬Æ¬±√ö
6√ü9¬≥_¬π¬Ø√Ω‚Äπlv¬≠4‚Äù‚Äô,C√ë<1≈°≈†‚ÄöY√ö√ò>¬ª¬∞"G√ï	¬∏¬†√º¬∏√±√ÄKC@≈†√õ[<¬ê√≠\d√ã&,U[Qb√ã√Ö¬¥¬§√ö√∫‚Äòdf√∞¬≠¬º6√≥≈°¬ªAv¬çY0√®√î√ó√£$\≈†23%‚Äú¬Å‚Ä°≈°¬£XF√ì‚Äπ¬¶¬°jL√á√ì√´√∏√∞√ª‚Ä∞√ü¬ù√§√±OCF6V√ú√í‚Äù'J√ç¬≥¬±+n¬Æ¬Å√èW√á√´√∏∆í_‚Äô√Ö5F¬∫√Ø‚Äòs¬∏√π'Zj√åx√ô√¨¬≥A‚Ä¢)‚Ä∫T!'O‚Äôf√§√Ñ¬Æow√óF√ßF‚Äù;Q‚Ñ¢1:[√ó√Ñ¬∂e√°√ß√∂√ì√öi‚Ä†¬ª√Ω√é<√Ñ√º≈∏#G:¬Æz‚ÄöUa√≠¬ç#‚Ä¶√ö‚Äô‚Ç¨11√°√É,√±4√è√õ√Ñ√É√óQ¬∑√ªe√Å‚Ñ¢o√Ω{¬æ√Ü$YIl?¬®√É‚Äô3√ª√ë@‚Ä∫¬≤¬∂F8√ñh¬∞Iysnq¬≥√ä√è¬≥≈ì$√õFq2e¬≤√î‚Ä†6l¬èZ¬¶;h	√æ¬Å[r-J√ò√Å√∑‚Äîo√ô‚Ä†√§
%L‚Ä∫¬ß!√°¬±=¬£	g√ü¬Ø¬≤6√õ9√é¬è√ù[D√é√¶	√ë‚Äö¬ùj√§D∆í‚Äú¬§√µ¬Å√ß¬¶‚Äú¬†O‚ÄöGm;√ë√£√ª√ü√ï¬è¬∑(o√±o?}√§+D√õ=√àe≈°3M‚Ä°√Ç\√¥√ÇBuÀÜ√à:‚Äû√öQÀÜ√≠`0t$P√Ç¬≤(
¬ù√è:¬ª√å7√ü¬®¬çf∆í¬¢√é>≈í√©\Ru¬º√•≈æfDNl>√åj√¶aG‚Ä∫¬©¬¶¬ù]¬Ω√ì¬ç¬∞√ìTm¬Øx¬∏~Àú5"\	√¶√∞[√´:W?}√ó¬∂i√ö¬è)¬ß>‚Äú@√®√É≈°≈†√™'‚Ä¶¬≠n¬ædK√ö6√í"‚Äö -√Ç√®¬πBJ¬¨
45 NULL null
45 NULL null
45 NULL null
45 NULL null
45 NULL null
46 NULL null
46 EOF 
